rails-snow-queen/app/assets/stylesheets/application.scss

```diff
@@ -65,4 +78,23 @@ header {
 
 .modal_errors {
   color: red;
+}
+
+.text-shadow {
+  text-shadow: 0.5px 0.5px 3px #4a4a4a;
+}
+
+.edit-button {
+  font-size: 20px;
+  border: 1px solid lightgray;
+  color: gray;
+}
+
+.font-size-35 {
+  font-size: 35px;
+}
+
+.currency-symbol {
+  font-size: 25px;
+  vertical-align: super;
```

Comment:
How come this needs special alignment?

Reply:
to format the '$' independent from the amount

Reply:
I was referring to the `vertical-align` specifically. What's the deal with `super`?